app/code/Magento/Checkout/view/frontend/web/js/sidebar.js

```diff
@@ -261,6 +261,9 @@ define([
                 $(document).trigger('ajax:removeFromCart', {
                     productIds: [productData['product_id']]
                 });
+                if (window.location.href === this.shoppingCartUrl) {
```

Comment:
Hi @ravi-chandra3197,

Could you fix the issue that was found by @engcom-Alfa, squash all changes to single commit and force push your changes?

I suggest to fix it like this:
```suggestion
                if (window.location.href.indexOf(this.shoppingCartUrl) === 0) {
```

Thank you!